content/reference/sql/version-control/dolt-sql-procedures.md

```diff
@@ -564,6 +564,46 @@ CALL DOLT_COMMIT('-a', '-m', 'committing all changes');
 CALL DOLT_MERGE('feature-branch', '--author', 'John Doe <johndoe@example.com>');
 ```
 
+## `DOLT_PATCH()`
+
+Get SQL statements on table diffs between revisions. This command is
```

Comment:
Docs look good. Just one suggestion for the very first sentence to make it as descriptive as possible and mention a concrete use case. 

```suggestion
Generate the SQL statements needed to patch a table (or all tables) from a starting revision to a target revision. This can be useful when you want to import data into Dolt from an external source, compare differences, and generate the SQL statements needed to patch the original source. This command is
```